### PR TITLE
Great books/project library

### PR DESCRIPTION
**Pull request by johannacatalinismith**
## Netlify link
https://libraryofgreatbooks.netlify.app/

## Collaborators
[github-username-annsig]

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  border: none;
+  border-radius: 30px;
+  padding: 16px 20px;
+  margin-right: 25px;
+  font-size: 16px;
+  line-height: 26px;
+  font-family: "Fira Sans";
+  font-weight: 500;
+  cursor: pointer;
+  transition: all 0.3s ease;
```
Comment (NellaMori) [inline style.css:104]: When you apply an effect to the button, like in this case the transition, is making all the UI around move, and this is to be avoided. A solution would be to wrap the button in a div and give the div a fixed height 👍

File: script.js [hunk 2]
```diff
@@ -4,403 +4,294 @@ of the properties make sense to filter on, and some to sort on.*/
...
-      'A dystopian novel set in a seemingly perfect society where young Jonas discovers the dark truth beneath the surface.',
-    image: './books-images/unknown.jpg'
+      "A dystopian novel set in a seemingly perfect society where young Jonas discovers the dark truth beneath the surface.",
+    image: "./books-images/unknown.jpg",
+  },
+];
+
+//Declare variables that refer to HTML elements
+const newestButton = document.getElementById("newestButton");
+const oldestButton = document.getElementById("oldestButton");
```
Comment (NellaMori) [inline script.js]: the naming of these variables could be improved to a better fitting description like `sortNewestBtn` and `sortOldestBtn` 👍  `Btn` is just shorter than `Button` and can be used interchangeably

File: style.css [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+  border-radius: 10px;
+}
+
+.book-title {
+  font-weight: bold;
+  margin-top: 20px;
+  font-family: "Fira Sans";
+}
+
+select {
```
Comment (NellaMori) [inline style.css:75]: tip to improve the UI: the arrow in the select element can have a little bit more margin to the right

**Review by NellaMori — CHANGES_REQUESTED** [review]
Hi Johanna, well done finishing your Library app 📚 Everything works pretty well, the only thing that needs to be added is app responsiveness (check the requirements on Notion).  I left some comments in the code, please update the app to make it responsive and you're good to go 🚀 thanks ⭐

File: style.css (hunk 1 above)
Comment (johannacatalinismith) [reply style.css:104]: Thanks! It's fixed!

File: script.js (hunk 2 above)
Comment (johannacatalinismith) [reply script.js]: Fixed!

File: style.css (hunk 3 above)
Comment (johannacatalinismith) [reply style.css:75]: Done :)

File: script.js [hunk 4]
```diff
@@ -4,403 +4,294 @@ of the properties make sense to filter on, and some to sort on.*/
...
+];
+
+//Declare variables that refer to HTML elements
+const sortNewestBtn = document.getElementById("newestButton");
+const sortOldestBtn = document.getElementById("oldestButton");
+const sortRatingBtn = document.getElementById("ratingButton");
+const randomBtn = document.getElementById("randomButton");
+
+
+//Books is the array. The "for loop" let's us go through the array of books to get the book(s) we need.
```
Comment (viktoria-olo) [inline script.js:195]: A general comment for the whole code: the comments are very clear and good at explaining what is going on. Well done!

File: script.js [hunk 5]
```diff
@@ -4,403 +4,294 @@ of the properties make sense to filter on, and some to sort on.*/
...
+  for (const book of books) {
+    booksDiv.innerHTML += renderBook(book);
+  }
+});
+
+sortOldestBtn.addEventListener("click", (event) => {
+  sortOldestBtn.classList.add("active") 
+  sortNewestBtn.classList.remove("active") 
+  sortRatingBtn.classList.remove("active")
+  randomBtn.classList.remove("active")
```
Comment (viktoria-olo) [inline script.js:261]: Could these perhaps have been turned into a function that removes "active" for each sort button?

**Review by viktoria-olo — COMMENTED** [review]
Well done! The code was clear and easy to follow. The comments were very helpful.

**Review by NellaMori — APPROVED** [review]

**Pull request closed without merging**